File: app/src/main/java/com/suxiunet/repair/base/baseui/SuXiuApplication.java
package com.suxiunet.repair.base.baseui;

import android.app.Application;

import com.suxiunet.data.DatasConstant;

/**
 * author : chenzhi
 * time   : 2018/01/14
 * desc   :
 */
public class SuXiuApplication extends Application {
    public static SuXiuApplication appContext;

    @Override
    public void onCreate() {
        super.onCreate();
        appContext = this;
//        DataConstant.Companion.setMApplication(this);
        DatasConstant.mApplication = this;
    }
}

File: app/src/main/java/com/suxiunet/repair/base/SuXiuApplication.kt
package com.suxiunet.repair.base

import android.app.Application

/**
 * author : chenzhi
 * time   : 2018/01/03
 * desc   : 
 */
class SuXiuApplication : Application() {
    companion object {
        lateinit var appContext: SuXiuApplication
    }

    override fun onCreate() {
        super.onCreate()
        appContext = this
    }
}